nemo/collections/asr/modules/transformer/transformer_generators.py

```diff
@@ -58,15 +59,20 @@ def __init__(
         max_sequence_length=512,
         max_delta_length=20,
         batch_size=1,
+        n_samples=1,
+        temperature=None,
     ):
         super().__init__()
         self.embedding = embedding
         self.decoder = decoder
         self.log_softmax = log_softmax
+        self.log_softmax.mlp.log_softmax = False
```

Comment:
Need make it work for any `log_softmax` modules, we can't assume that all those modules have `log_softmax.mlp.log_softmax`